src/txdb.cpp

```diff
@@ -758,25 +763,25 @@ CacheConfig CacheSizeCalculations(int64_t _nTotalCache)
     _nTotalCache -= cache.nBlockTreeDBCache;
     if (BLOCK_DB_MODE == LEVELDB_BLOCK_STORAGE)
     {
-        // use up to 5% for the level db block cache but no bigger than 256MB
-        cache.nBlockDBCache = _nTotalCache * 0.05;
+        // use up to 10% for the level db block cache but no bigger than 256MB
```

Comment:
In the code seems the code has been rised from 256 to 1024

Reply:
done